src/copy.c

```diff
@@ -335,13 +368,28 @@ timescaledb_CopyFrom(CopyChunkState *ccstate, List *range_table, Hypertable *ht)
 
 				/* OK, store the tuple and create index entries for it */
 				heap_insert(resultRelInfo->ri_RelationDesc, tuple, mycid, hi_options, bistate);
```

Comment:
I believe this needs to use `table_tuple_insert` or otherwise we won't invoke the right insert method for the table access method.